frontend/test/pytest/test_callback.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test callbacks"""
+
+import pennylane as qml
+
+from catalyst.pennylane_extensions import callback
+
+
+def test_callback_no_returns_no_params(capsys):
+    """Test callback no parameters no returns"""
+
+    @callback
+    def my_callback() -> None:
+        print("Hello erick")
+
+    @qml.qjit
+    def cir():
+        my_callback()
+        return None
+
```

Comment:
Might be nice to assert that nothing is printed during compilation as well
```suggestion

    captured = capsys.readouterr()
    assert captured.out.strip() == ""

```

Reply:
https://github.com/PennyLaneAI/catalyst/pull/540/commits/953377f38c07c477ef596aa958747ca1cafebf18